1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,26 @@
 function readingList(books) {
-  // Write your code here...
+  const readingList = document.createElement(`ul`);
```

Comment:
Do you need to create this? Have you tried running the test command.

I get 

```plaintext
received value must be an HTMLElement or an SVGElement.
    Received has value: null
```

Why do you think that is?

Reply:
Hint - check the developer tools and look at the elements!